compute_endpoint/globus_compute_endpoint/executors/high_throughput/executor.py

```diff
@@ -1,936 +1,3 @@
-"""HighThroughputExecutor builds on the Swift/T EMEWS architecture to use MPI for fast
-task distribution
+from globus_compute_endpoint.engines import HighThroughputEngine
```

Comment:
This is to give end-users time to transition?  Suggest then that we implement a deprecation notice wrapper.  Perhaps:
```python
# Prefix an underscore to discourage using _this_ module to find the HighThroughtputEngine
from globus_compute_endpoint.engines import HighThroughputEngine as _HighThroughputEngine

def HighThroughputExecutor(*args, **kwargs):
    import warnings
    warnings.warn("Deprecated ... please use 'engines...Engine' instead")
    return _HighThroughputEngine(*args, **kwargs)

```